### ISSUE TITLE
Just downloaded

### ISSUE THREAD

**Issue by Emanuele20** [StudioJunkyard/LibreCMS#3]
Hi thx for sharing ur work but...
uploaded to my web linux server open my directory (http://www.myweb/libre) and see: "mod_rewrite" must be available and enabled for LibreCMS to function correctly.
On my server mod_rewrite works...
Maybe i miss something to do before?
Kindly Regards
Emanuele

*Ok now appear ERR_TOO_MANY_REDIRECTS

**Comment by StudioJunkyard** [comment]
The link you provided doesn't work, there's no TLD (.com/.net). Also when opening the first time, make sure that `core/config.ini` isn't present, and delete it if it does. The installer will create it.

I'm also wondering if it's because your trying to run it in a folder, rather than in the root of the domain. Though, I don't think this should be an issue, as I run it locally in a folder.

**Comment by Emanuele20** [comment]
Hi, thx for ur availability ....try delete and re-upload on my server ur project, deleting config.ini ... now appear this: Warning: parse_ini_file(config.ini): failed to open stream: No such file or directory in /web/htdocs/www.emanueleosvaldopivetta.it/home/develop/core/db.php on line 7....Just 1 question....can i install manually? Creating mysql table in mysql db and making configuration manually? the domain above is just an example to make understand that ur project is in a directory...

**Comment by StudioJunkyard** [comment]
Yeah, you can, the .sql file is called "libre.sql", and is located in the /core folder. The default username:password is root:dev 
Then as you would assume edit the config.ini to reflect that.

**Comment by StudioJunkyard** [comment]
If your having issue's with the installer, I may have to look into, and refine it some more. So far I've only used it for my own clients.

Your the first to give feed back with this version thus far.

Thanks for that BTW.

Just because it works fine for me, doesn't mean others can work it out.

**Comment by Emanuele20** [comment]
Really really thx for ur kindly answer and for ur availability. I cannot
give u domain access cuz domain isn't mine! Now here(always into a
directory of the root) is working a custom cms in mysql syntax and must be
update to pdo or mysqli (pdo my best choice)...so, i ve to find a way out!
I'd like to use ur cms cuz is "little" and for sure could be mod to new
possibility...like plugin or simple data entry or js effect...

2016-09-13 10:21 GMT+02:00 Dennis Suitters <notifications@github.com>:

**Comment by StudioJunkyard** [comment]
I understand why can't give domain access, that's ok.

LibreCMS uses PDO by default, which as you know at present is the best option for database interaction.

I'm open to modding things, though I'm trying to stay away from a plugin eco-system. Plugins are one (among many issues) of WordPresses biggest security issues (besides the fact they don't use PDO, or safe method of storing passwords).

There are some things I want to rewrite with LibreCMS. If you've looked at the source, adding new content pages isn't easy, and as it is, I'm about to add a new content type to satisfy some SEO stuff I've been reading about in Reddit to do with old, and out of stock inventory. Anyway, to add a new content type, and page for it, I have to manually modify the router, which as you would imagine a Pain In The A**. I'd like to be able to allow the additions of new pages within the Pages section of the Admin Area.

Plus a few little things, like maybe if the time is right converting to Bootstrap 4 for the admin if it's released at the time.

If you haven't, check out the wiki, I'll be adding content in there too to explain how pages work, as they are linked to the wiki from each page in the Admin.

Let me know if you get things working, I'm sorry other than suggestions, there's not much I can do. But if you can let me know what error messages you get in the your logs or on screen, I can try and replicate them here.

Actually, if you can, try and set LibreCMS up on your local machine.

**Comment by StudioJunkyard** [comment]
I just pushed an update, which includes, being able to change pages, and contents meta-robots options, so old content, or badly performing pages can be pulled from being indexed by search engines.

I've also added, on request from an upcoming client, being able to add options for content items, mainly for products, so colour, or size options can be added.

Because of the above, there are some database additions as well.

I checked out the installer on my local machine with no issues, as well as on my live server, also without issues.

**Comment by StudioJunkyard** [comment]
Nearly forgot, also started creating code for the addition of using AMP, as that's apparently going to be a thing as well. I've done most of the code to allow it to work, just need to work out tags for the templating.

**Comment by Emanuele20** [comment]
Hello! Downloaded with ur update but always this error in log...
["mod_rewrite" must be available and enabled for LibreCMS to function correctly.]

For available i think im ok...look this another code: 2 files
.htaccess -- RewriteEngine On
         RewriteRule ^.*$ rewrite.php --
rewrite.php -- <?php echo "Mod_rewrite is activated!"; ?> --

Thx for ur kindly best regards Emanuele

**Comment by StudioJunkyard** [comment]
hm, ok, it should work in a folder, looks like something I need to look at.

I was going to suggest to check which version of PHP your using, 5.5.9 is the lowest supported. What reminded me of this, is that on my server with VirtualMin, it sometimes sets the PHP version to use to 5.4, which LibreCMS uses functions not available in 5.4, and it will throw errors.

**Comment by Emanuele20** [comment]
My php version is 5.5.36 this could be the problem....changed in 5.6 right now...let me try again installing from in a directroy....ve i to change the .htaccess or not?

**Comment by StudioJunkyard** [comment]
the default .htaccess should only need the https section changed. I've tried to have as much auto stuff as possible.

**Comment by StudioJunkyard** [comment]
Hi @Emanuele20 just wondering how you were going with LibreCMS, I've just pushed a new update which includes some adjustments to the installer, one of which is to check if the config.ini is present and if so it is unlinked (deleted), and written as new. I also tried installing into a folder on my localhost, and didn't have issue's doing that. Let me know how your going if your still interested.
Hope you've had an awesome Christmas break.
